app/models/basket.rb

```diff
@@ -2,4 +2,6 @@ class Basket < ApplicationRecord
   belongs_to :user
   has_many :subscriptions, dependent: :destroy
   has_many :basket_items, dependent: :destroy
+  validates :user_id, :price, :title, :availability, :description, presence: true
+  validates :title, :description, uniqueness: true
```

Comment:
Why should description also be unique?

Reply:
My concern would be a user creating 100s of the same basket. Perhaps, the uniqueness should be that a title/description pair should be unique.

Reply:
validates :title, uniqueness: { scope: :description }. Something like this be better?

Reply:
Yes I prefer this version, could you update?